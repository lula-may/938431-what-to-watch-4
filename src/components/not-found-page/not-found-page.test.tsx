import * as React from "react";
import {Router} from "react-router-dom";
import * as renderer from "react-test-renderer";

import NotFoundPage from "./not-found-page";
import history from "../../history";

describe(`NotFoundPage`, () => {
  it(`should render correctly 404 Page`, () => {
    const tree = renderer.create(
        <Router history={history}>
          <NotFoundPage/>
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
