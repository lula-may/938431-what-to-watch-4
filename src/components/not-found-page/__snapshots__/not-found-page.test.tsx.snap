// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render correctly 404 Page 1`] = `
<div
  className="user-page"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    404
  </h1>
  <h2>
    Page not found
  </h2>
  <a
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "#c9b37e",
      }
    }
  >
    Go to main page
  </a>
</div>
`;
